test/test_functionalization.py

```diff
@@ -101,6 +102,56 @@ def f(x):
             return z2
         self.assert_functionalization(f, torch.ones(4))
 
+    def test_view_clone_view_inplace(self):
+        def f(input):
```

Comment:
could you please add a comment for what we are testing here? (what was the bug and how does the fix resolve it ...)